### PR TITLE
Clarify helm version supported and other minor changes.

### PR DESCRIPTION
**Pull request by skonto**
### What changes were proposed in this pull request?
Make specific the Helm version supported.

### Why are the changes needed?
Right now instructions dont specify a Helm version. Since we dont support Helm 3 yet we should make clear what version is supported, otherwise things might break at the user side.

### CODE REVIEW

**Comment by skonto** [comment]
@agemooij I will update the PR here to improve the page here (https://cloudflow.io/docs/current/get-started/setup-gke-cluster.html).

File: docs-source/docs/modules/get-started/pages/setup-gke-cluster.adoc [hunk 1]
```diff
@@ -5,9 +5,9 @@
 
 include::ROOT:partial$include.adoc[]
 
-Detailed instructions on how to install a GKE cluster with Cloudflow are described in the https://github.com/lightbend/cloudflow/tree/master/installer[Cloudflow installer section] of the GitHub repo. 
+Detailed instructions on how to install a GKE cluster with Cloudflow are described in the https://github.com/lightbend/cloudflow/tree/master/installer[Cloudflow installer section] of the GitHub repo.
 
-To install a new cluster, clone the https://github.com/lightbend/cloudflow/[repo] in your local system, go to the `installer/` folder and follow these instructions.
+To install a new cluster, clone the https://github.com/lightbend/cloudflow/[repo] in your local system by selecting the latest release branch or download the latest release file from https://github.com/lightbend/cloudflow/releases[here], go to the `installer/` folder and follow these instructions shown next.
```
Comment (agemooij) [inline docs-source/docs/modules/get-started/pages/setup-gke-cluster.adoc]:
```suggestion
To install a new cluster, clone the https://github.com/lightbend/cloudflow/[repo] in your local system by selecting the latest release branch or download the latest release file from https://github.com/lightbend/cloudflow/releases[here], go to the `installer/` folder and follow the instructions below.
```

File: docs-source/docs/modules/get-started/pages/setup-gke-cluster.adoc [hunk 2]
```diff
@@ -16,21 +16,24 @@ NOTE:: This can be done once you have a valid Google account and have access to
 * Create a GKE cluster with a specific name `./create-cluster-gke.sh <cluster-name>`
 * Install Cloudflow in the designated cluster `./install-gke.sh  <cluster-name>`
 
+NOTE:: This step uses Helm which can be installed from https://helm.sh/docs/intro/install[here]. Currently only Helm 2 is supported while Helm 3 support is WIP.
```
Comment (agemooij) [inline docs-source/docs/modules/get-started/pages/setup-gke-cluster.adoc]: @skonto I would point them at the V2-specific installation page instead, e.g. https://v2.helm.sh/docs/using_helm/#installing-helm.

**Comment by skonto** [comment]
@agemooij updated the PR pls merge.

File: docs-source/docs/modules/get-started/pages/setup-gke-cluster.adoc [hunk 3]
```diff
@@ -5,9 +5,9 @@
 
 include::ROOT:partial$include.adoc[]
 
-Detailed instructions on how to install a GKE cluster with Cloudflow are described in the https://github.com/lightbend/cloudflow/tree/master/installer[Cloudflow installer section] of the GitHub repo. 
+Detailed instructions on how to install a GKE cluster with Cloudflow are described in the https://github.com/lightbend/cloudflow/tree/master/installer[Cloudflow installer section] of the GitHub repo.
 
-To install a new cluster, clone the https://github.com/lightbend/cloudflow/[repo] in your local system, go to the `installer/` folder and follow these instructions.
+To install a new cluster, clone the https://github.com/lightbend/cloudflow/[repo] in your local system by selecting the latest release branch or download the latest release file from https://github.com/lightbend/cloudflow/releases[here], go to the `installer/` folder and follow the instructions bellow.
```
Comment (agemooij) [inline docs-source/docs/modules/get-started/pages/setup-gke-cluster.adoc]:
```suggestion
To install a new cluster, clone the https://github.com/lightbend/cloudflow/[repo] in your local system by selecting the latest release branch or download the latest release file from https://github.com/lightbend/cloudflow/releases[here], go to the `installer/` folder and follow the instructions below.
```

**Comment by agemooij** [comment]
@skonto I found one last typo but I can't commit it to your branch so I will just merge it and then fix master just to expedite things

**Pull request merged**